Fix inconsistency in root key migration paragraph (#61)
theupdateframework/specification: tuf-spec.md

   role.  When replacing root keys, an application will sign the new root.json
   file with both the new and old root keys. Any time such a change is
   required, the root.json file is versioned and accessible by version number,
   e.g., 3.root.json.

   Clients that have outdated root keys can update to the latest set of trusted
   root keys, by incrementally downloading all intermediate root metadata

>    files, and verifying that each root metadata at a given version is signed by

Reviewer: ```suggestion
   files, and verifying that each current version of the root metadata is signed by
```

>    a threshold of keys defined in the previous version of root, and a threshold

Reviewer: ```suggestion
   a threshold of keys specified by its immediate predecessor as well as a threshold
```

>    of keys defined in that given version of root.

Reviewer: ```suggestion
   of keys specified by itself.
```

   For example, if there is a 1.root.json that has threshold 2 and a
   2.root.json that has threshold 3, 2.root.json MUST be signed by at least 2
   keys defined in 1.root.json and at least 3 keys defined in 2.root.json.

>    The client starts the root keys update update process with the latest

Reviewer 2: ```suggestion
   The client starts the root key update process with the latest
```

   role.  When replacing root keys, an application will sign the new root.json
   file with both the new and old root keys. Any time such a change is
   required, the root.json file is versioned and accessible by version number,
   e.g., 3.root.json.

   Clients that have outdated root keys can update to the latest set of trusted
   root keys, by incrementally downloading all intermediate root metadata
   files, and verifying that each current version of the root metadata is
   signed by a threshold of keys specified by its immediate predecessor as well
   as a threshold of keys specified by itself.
   For example, if there is a 1.root.json that has threshold 2 and a
   2.root.json that has threshold 3, 2.root.json MUST be signed by at least 2
   keys defined in 1.root.json and at least 3 keys defined in 2.root.json. The
   client starts the root key update process with the latest version of root

>    metadata available on the client, and stops when no newer root metadata is

Reviewer: ```suggestion
   metadata available on the client, and stops when no version N+1 (where N is the latest trusted version) of the root metadata is
```
